app/cdap/components/WrangleHome/services/getWidgetData.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { getCategorizedConnections } from 'components/Connections/Browser/SidePanel/apiHelpers';
+import {
+  fetchAllConnectorPluginProperties,
+  fetchConnectionDetails,
+  fetchConnectors,
+  getMapOfConnectorToPluginProperties,
+  getSelectedConnectorDisplayName,
+} from 'components/Connections/Create/reducer';
+import WidgetSVG from 'components/WrangleHome/Components/WidgetSVG';
+import { getCategoriesToConnectorsMap } from 'components/WrangleHome/Components/WidgetSVG/utils';
+import { AddConnectionIcon } from 'components/WrangleHome/Components/WrangleCard/iconStore/AddConnectionIcon';
+import { ImportDataIcon } from 'components/WrangleHome/Components/WrangleCard/iconStore/ImportDataIcon';
+import { ImportDatasetIcon } from 'components/WrangleHome/Components/WrangleCard/iconStore/ImportDatasetIcon';
+import {
+  IConnectorArray,
+  IConnectorDetailPayloadArray,
+} from 'components/WrangleHome/Components/WrangleCard/types';
+import React from 'react';
+
```

Comment:
This needs a jsDoc comment and it doesn't need a callback passed to it - just return the widget data

Reply:
Done